tests/unit/commands/deploy/test_guided_context.py

```diff
@@ -1,5 +1,5 @@
 from unittest import TestCase
-from unittest.mock import patch, call, ANY, MagicMock
+from unittest.mock import patch, call, ANY, MagicMock, Mock
```

Comment:
nit: I can't find `Mock` being used anywhere in this file. Do we need to import it?

Reply:
Removed. Thanks!